blog/2023-11-05-databend-weekly-118.md

```diff
@@ -0,0 +1,110 @@
+  - name: JackTan25
+  - name: lichuang
+  - name: PsiACE
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zenus
+  - name: zhang2014
+  - name: ZhiHanZ
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's New
+
+Stay informed about the latest features of Databend.
+
+### MERGE INTO Now Supports Distributed Execution
+
+The `MERGE INTO` statement performs insert, update, or delete operations on rows in the target table based on specified conditions and matching criteria using data from the specified source.
+
+In Databend, the `MERGE INTO` statement now supports distributed execution, which enables more efficient data integration capabilities. The physical plan for distributed execution is shown in the following diagram:
+
+![Image: Databend Copy Into](../static/img/blog/merge-into-physical-plan.png)
+
+Please [contact the Databend team](https://www.databend.com/contact-us) if you would like to learn more, or check out the resources below:
+
+- [PR #13151 | feat: support distributed merge_into](https://github.com/datafuselabs/databend/pull/13151)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Incorporate Data Version Control into Your Analytics Workflow with LakeFS
+
+LakeFS provides a solution for sharing and collaborating on data processing by empowering object storage with Git-like operations. It offers a versioning scheme that provides logically consistent views of data, meaningful branch names, and commit messages for modern data workflows, as well as integrated solutions for data and document unification.
+
+Databend not only provides a distributed, elastic, and scalable query engine with high performance but also supports multi-version storage similar to Git. This allows for easy querying, cloning, and restoring of data at any point in time.
+
+By combining LakeFS's data versioning capabilities with Databend's Git-like data management capabilities, data integrity can be ensured, and efficient and highly available consistent views can be provided for data analysis workflows.
+
+If you are interested in learning more, please check out the resources below:
+
+- [Blog | Incorporate Data Version Control into Your Analytics Workflow](https://databend.rs/blog/2023-11-02-databend-with-lakefs)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Added support for JSON operators: `?`, `?|`, and `?&`.
+- Added support for permissions management on `STAGE` and `UDF`.
```

Comment:
Could you share the PR link?

Reply:
https://github.com/datafuselabs/databend/pull/13445